packages/cloud-cognitive/src/components/EditFullPage/EditFullPage.mdx

```diff
@@ -0,0 +1,227 @@
+import { Story, ArgsTable, Source, Canvas } from '@storybook/addon-docs';
+import {
+  getStoryId,
+  CodesandboxLink,
+} from '../../global/js/utils/story-helper';
+import { CreateFullPage } from '.';
```

Comment:
Can we double check references to `CreateFullPage` are updated to `EditFullPage`?